spotbugs-tests/src/test/java/edu/umd/cs/findbugs/ba/ValueRangeDataflowTest.java

```diff
@@ -0,0 +1,253 @@
+package edu.umd.cs.findbugs.ba;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import edu.umd.cs.findbugs.AbstractIntegrationTest;
+
+public class ValueRangeDataflowTest extends AbstractIntegrationTest {
+
+    private final ByteArrayOutputStream outContent = new ByteArrayOutputStream();
+    private final PrintStream originalOut = System.out;
+
+    @Before
+    public void setUpStreams() {
+        System.setOut(new PrintStream(outContent));
+    }
+
+    @After
+    public void restoreStreams() {
+        System.setOut(originalOut);
+    }
+
+    @Test
+    public void test1() {
+        System.setProperty("dataflow.classname", "edu.umd.cs.findbugs.ba.ValueRangeDataflow");
+        System.setProperty("dataflow.printcfg", "true");
+        performAnalysis("ValueRangeDataflow.class");
+        String[] outputLines = outContent.toString().split(System.lineSeparator());
+        checkOutput(outputLines);
+    }
+
+    private void checkOutput(String[] outputLines) {
+        for (int i = 0; i < outputLines.length; ++i) {
+            if ("Method: ValueRangeDataflow.eq(int)".equals(outputLines[i])) {
+                checkEQ(outputLines, i + 4);
```

Comment:
`i + 4` is repeated in each conditional. Perhaps this could be refactored?

Reply:
This got fixed in https://github.com/spotbugs/spotbugs/pull/2073/commits/ae3d629a9e2af9d4a2319727f131de62b86b5e12, so this conversation can be resolved.